src/blocks/HeaderSlider/HeaderSlider.tsx

```diff
@@ -4,7 +4,7 @@ import {MobileContext} from '../../context/mobileContext';
 import {HeaderSliderBlockProps, SliderType} from '../../models';
 import {block} from '../../utils';
 import Header from '../Header/Header';
-import {SliderBlock} from '../index';
+import {SliderNewBlock as SliderBlock} from '../unstable';
```

Comment:
if it is a breaking change I think we need to delete old slider and use new like as the main one

Reply:
I replaced SliderBlock with the new one and renamed old one as SliderOldBlock